setup.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+'''
+Setup script for python build system
+'''
+import os
+from setuptools import setup, find_packages
+
+HERE = os.path.dirname(__file__)
+REPO_URL = 'https://github.com/jitsuin-inc/archivist-samples/'
+NAME = "jitsuin-archivist-samples"
+
+with open(os.path.join(HERE, 'README.md')) as FF:
+    DESC = FF.read()
+
+with open(os.path.join(HERE, 'requirements.txt')) as FF:
+    requirements=[f"{line.strip()}" for line in FF]
+
+setup(
+    name=NAME,
+    author="Jitsuin Inc.",
+    author_email="support@jitsuin.com",
+    description="Jitsuin Archivist Examples",
+    long_description=DESC,
+    long_description_content_type="text/markdown",
+    url=REPO_URL,
+    packages=find_packages(exclude=( "examples", "unittests", )),
+    platforms=['any'],
+    classifiers=[
+        'Development Status :: 3 - Alpha', #pre-delivery
```

Comment:
Is there a SPDX identifier we can use here as well or instead of ?

Reply:
The standard pypi license iclassifier is from here: https://gist.github.com/nazrulworld/3800c84e28dc464b2b30cec8bc1287fc

                    License :: OSI Approved :: MIT License